src/Normalizer/ContentEntityNormalizer.php

```diff
@@ -53,17 +50,68 @@ class ContentEntityNormalizer extends NormalizerBase implements DenormalizerInte
   protected $format = array('json');
 
   /**
+   * @var int
+   */
+  protected $entity_id = null;
```

Comment:
As noted below, I'd prefer to have these parameters passed directly as arguments to the methods where they are used.